common-lib/src/main/scala/com/gu/mediaservice/lib/collections/CollectionsManager.scala

```diff
@@ -38,4 +42,15 @@ object CollectionsManager {
 
   // We could use `ValidationNel`s here, but that's overkill
   def isValidPathBit(s: String) = if (s.contains(delimiter)) false else true
+
+  val collectionColours = Map(
+    "g2"           -> "#000000",
+    "observer"     -> "#006f94",
+    "culture"      -> "#d1008b",
+    "film & music" -> "#b1532f"
```

Comment:
`path` and `pathId` are always in lowercase. `description` can be in WobblyCase.
[See here for more](https://github.com/guardian/grid/pull/1612)